### PR TITLE
New package: atsuover.Garcello version 1aec0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112006&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/a/atsuover/Garcello/1aec0/atsuover.Garcello.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: atsuover.Garcello
+PackageVersion: 1aec0
+PackageLocale: en-US
+Publisher: atsuover
+PackageName: Smoke 'Em Out Struggle [FULL WEEK] | VS Garcello
+License: Apache 2
```
Comment (stephengillie) [inline manifests/a/atsuover/Garcello/1aec0/atsuover.Garcello.locale.en-US.yaml:9]:
```suggestion
PackageName: Smoke 'Em Out Struggle
PackageUrl: https://gamebanana.com/mods/166531
License: Creative Commons 4.0
```

**Comment by BrandonWanHuanSheng** [comment]
#111634

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
https://github.com/microsoft/winget-pkgs/issues/107639

**Pull request merged**